specs/windows/nt_info.table

```diff
@@ -0,0 +1,23 @@
+table_name("nt_info") description(
+    "Display basic NT domain information of a Windows machine.")
+    schema([
+      Column("name", TEXT, "The label by which the object is known."),
+      Column("client_site_name",
+             TEXT,
+             "The name of the site where the domain controller is configured."),
```

Comment:
Why does the name of the configuration site for the domain controller has a name `client`?